### PR TITLE
Test get command

### PR DESCRIPTION
**Pull request by alemagio**

### CODE REVIEW

File: test/get-command.test.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  expect(getCommand('npm', 'install')).toBe('npm install')
+})
+test('should return correct command without args for npm', () => {
+  expect(getCommand('npm', 'test')).toBe('npm run test')
+})
+test('should return correct command with args for npm', () => {
+  expect(getCommand('npm', 'test', 'my args')).toBe('npm run test -- my args')
+})
+
+test('should return correct install command for other package managers', () => {
```
Comment (fox1t) [inline test/get-command.test.ts]:
```suggestion
test('should return correct install command for pnpm', () => {
```

File: test/get-command.test.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  expect(getCommand('pnpm', 'install')).toBe('pnpm install')
+})
+test('should return correct command without args for other package managers', () => {
```
Comment (fox1t) [inline test/get-command.test.ts]:
```suggestion
test('should return correct command without args for pnpm', () => {
```

File: test/get-command.test.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  expect(getCommand('pnpm', 'test')).toBe('pnpm test')
+})
+test('should return correct command with args for other package managers', () => {
```
Comment (fox1t) [inline test/get-command.test.ts]:
```suggestion
test('should return correct command with args for pnpm', () => {
```

**Review by fox1t — CHANGES_REQUESTED** [review]

File: test/get-command.test.ts (hunk 1 above)
Comment (alemagio) [reply test/get-command.test.ts]: technically the same behaviour will happen with any other package managers 
anyway, let's move on

**Pull request merged**